bodhi/client/__init__.py

```diff
@@ -432,9 +464,10 @@ def edit(user, password, url, **kwargs):
 @click.option('--mine', is_flag=True, help='Show only your updates')
 @staging_option
 @url_option
+@debug_option
 @add_options(pagination_options)
 @handle_errors
-def query(url, mine=False, rows=None, **kwargs):
+def query(url, mine=False, rows=None, debug=False, **kwargs):
```

Comment:
Let's not define the default for ```debug``` in two places. If you move this to be after ```url```, you don't need to give it a default.

Reply:
Will fix